src/test/java/edu/ksu/canvas/attendance/services/AttendanceSectionServiceUTest.java

```diff
@@ -88,5 +97,28 @@ public void getSectionsByCourse_HappyPath() {
         
         assertThat(actualSections, contains(firstSection, secondSection));
     }
+
+    @Test
+    public void resetAttendanceAssignment_HappyPath() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00100)